Cannot handle large zip files

I'm seeing the following error in the pdp logs:

```python
Error handling request
[2016-11-05 07:13:40 +0000] [26765] [ERROR] Error handling request
Traceback (most recent call last):
  File "/home/websites/tools.pacificclimate.org/data_portal-2.3.6/pdp/pyenv/local/lib/python2.7/site-packages/gunicorn/workers/async.py", line 112, in handle_request
    for item in respiter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.3.6/pdp/pyenv/local/lib/python2.7/site-packages/pdp-2.3.6-py2.7.egg/pdp/error.py", line 62, in __call__
    start_response(status, response_headers, sys.exc_info())
  File "/home/websites/tools.pacificclimate.org/data_portal-2.3.6/pdp/pyenv/local/lib/python2.7/site-packages/gunicorn/http/wsgi.py", line 241, in start_response
    reraise(exc_info[0], exc_info[1], exc_info[2])
  File "/home/websites/tools.pacificclimate.org/data_portal-2.3.6/pdp/pyenv/local/lib/python2.7/site-packages/pdp-2.3.6-py2.7.egg/pdp/error.py", line 56, in __call__
    for block in response_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.3.6/pdp/pyenv/local/lib/python2.7/site-packages/ga_wsgi_client/__init__.py", line 41, in __call__
    for chunk in app_iter:
  File "/home/websites/tools.pacificclimate.org/data_portal-2.3.6/pdp/pyenv/local/lib/python2.7/site-packages/pydap/responses/aaigrid/__init__.py", line 36, in ziperator
    z.writestr(name, ''.join([x for x in responder]))
  File "/usr/lib/python2.7/zipfile.py", line 1145, in writestr
    self._writecheck(zinfo)
  File "/usr/lib/python2.7/zipfile.py", line 1040, in _writecheck
    raise LargeZipFile("Zipfile size would require ZIP64 extensions")
LargeZipFile: Zipfile size would require ZIP64 extensions
```

Looks like we're probably getting requests for a large number of timesteps, then when this extension tries to create one grid file per timestep, it tries to create a zipfile that's larger than 4 GB. [Enabling the 64bit LargeZipFile extension](https://docs.python.org/2/library/zipfile.html#zipfile.ZipFile) should fix this.

--- a/.travis.yml
+++ b/.travis.yml
@@ -4,7 +4,7 @@ before_install:
     - sudo apt-get update -qq
     - sudo apt-get install -qq python-gdal python-numpy libhdf5-serial-dev
 install:
-    - pip install -r requirements.txt -r test_requirements.txt -i http://tools.pacificclimate.org/pypiserver/
-    - pip install -i http://tools.pacificclimate.org/pypiserver/ .
+    - pip install -r requirements.txt -r test_requirements.txt -i https://pypi.pacificclimate.org/simple/
+    - pip install -i https://pypi.pacificclimate.org/simple/ .
 script:
     - py.test

--- a/src/pydap/responses/aaigrid/__init__.py
+++ b/src/pydap/responses/aaigrid/__init__.py
@@ -29,7 +29,7 @@ def ziperator(responders):
     '''
     with SpooledTemporaryFile(1024*1024*1024) as f:
         yield 'PK' # Response headers aren't sent until the first chunk of data is sent.  Let's get this repsonse moving!
-        z = ZipFile(f, 'w', ZIP_DEFLATED)
+        z = ZipFile(f, 'w', ZIP_DEFLATED, True)
 
         for name, responder in responders:
             pos = 2 if f.tell() == 0 else f.tell()

